src/providers/SignatureProvider.tsx

```diff
@@ -2,47 +2,80 @@ import { localStorageKeys, useLocalStorageState } from "hooks/useLocalStorage";
 import React, { createContext, CSSProperties, FC, useContext, useState } from "react";
 
 interface InputElementOptions {
+    type: "name" | "email" | "title" | "phone" | "linkedin" | "company";
```

Comment:
Looking good! Now that things are cleaner and working - lets rename `type` to `variant` 

As @ramirezStefano mentioned, it may be confusing as it is a reserved word in typescript